pkg/platform/kube/platform.go

```diff
@@ -1665,6 +1670,10 @@ func (p *Platform) enrichHTTPTriggerIngresses(ctx context.Context,
 				encodedIngressMap["host"] = renderedIngressHost
 			}
 		}
+
+		if _, ingressPathTypeFound := encodedIngressMap["pathType"].(networkingv1.PathType); !ingressPathTypeFound {
+			encodedIngressMap["pathType"] = networkingv1.PathTypeImplementationSpecific
```

Comment:
I havent seen you are using it anywhere. you need to add this special handling on `functionconfig/types.go` where for `GetFunctionIngresses` function. it is being used by the lazy client which then returns list of ingresses to create for a function.

Reply:
Dashboard enriches that is true.
But user can edit or change his function config via kubectl.
Also remember that controller also enrich from platform config.
Bottom line, dashboard enriches the pathType on ingress attributes and controller uses it using the function mentioned above.